README.md

```diff
@@ -11,12 +11,22 @@ See [examples](examples/) for how to use the PI.
 
 - libjudy-dev
 - libreadline-dev
+- libtool-bin
+- valgrind
 
 ## Building p4runtime.proto
 
 To include `p4runtime.proto` in the build, please run `configure` with
 `--with-proto`.
 
+```
+./autogen.sh
+./configure [--with-proto] [--with-bmv2]
+make
+sudo make check
```

Comment:
`sudo` should not be needed here

Reply:
nope, SUDO make check is necessary. or nanomsg ipc tests will fail

Reply:
This is not correct, unless you have some left-over ipc files which were created with root privileges. That is because nanomsg doesn't unlink the files when `nn_close` is called (see https://github.com/nanomsg/nanomsg/issues/666).
I suggest you try running `sudo rm -rf /tmp/*.ipc`. After that you will be able to run `make check` without `sudo` any number of times.

Reply:
Yes, you are right. I remove *.ipc and it works.
I have corrected the line, could you please review again, thank you!